### PR TITLE
Bug/repair warnings for thrashbar

### PR DESCRIPTION
**Pull request by OniWarrior**
In this ticket, I repaired several defined but never used warnings:

./src/components/pages/Gamemode/Thrashbar.jsx
Line 1:17: 'useState' is defined but never used no-unused-vars
Line 1:27: 'useEffect' is defined but never used no-unused-vars
Line 1:38: 'Component' is defined but never used no-unused-vars
Line 3:20: 'Button' is defined but never used no-unused-vars

./src/components/pages/GamePlay/GamePlayMain.js
Line 3:10: 'connect' is defined but never used

In Thrashbar.jsx, there was a useState,useEffect,and Component import that I removed that were never implemented on #1.  I removed the Button import on #3 because it was never used.

In GamePlayMain, that warning is outdated. Somebody else has implemented connect.

On galleryReducer.js I commented out what I assume where suppose to be comments on #4 and #29. I'm making the assumption that when changes to the reducer were being made, these accidentally slipped through.

Trello card: https://trello.com/c/YoMUWFUK/602-defined-but-never-used-e

PR submission video: https://drive.google.com/file/d/1lyC2sd09PnqdlQRQVwCZJplpGf5WSXWQ/view?usp=sharing

### CODE REVIEW

File: src/components/pages/Gamemode/Thrashbar.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect, Component } from 'react';
+import React from 'react';
```
Comment (Yonathan-Admasu728) [inline src/components/pages/Gamemode/Thrashbar.jsx:2]: great job! spelling out, that this is a class component not a functional component.

File: src/components/pages/Gamemode/Thrashbar.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 import { Row, Col, Button } from 'antd';
 import { connect } from 'react-redux';
```
Comment (Yonathan-Admasu728) [inline src/components/pages/Gamemode/Thrashbar.jsx:4]: good catch! the fact that somebody already implemented connect.

**Review by Yonathan-Admasu728 — DISMISSED** [review]
Excellent! attention to detail. good catch on "connect" being already implemented and also pointing out that we don't need "useState" hook in a class component.

**Review by MichaelMcFaddenjr — DISMISSED** [review]
Nice work removing the unused imports and cleaning up the code!!

**Comment by OniWarrior** [comment]
I removed the merging conflicts

**Review by MichaelMcFaddenjr — APPROVED** [review]
Nice work. See my inline comments for more specific feedback

**Review by Yonathan-Admasu728 — APPROVED** [review]

**Pull request merged**